makefile

```diff
@@ -75,3 +78,13 @@ AUTHORS:
 	@$(file >$@,# This file lists all individuals having contributed content to the repository.)
 	@$(file >>$@,# For how it is generated, see `make AUTHORS`.)
 	@echo "$(shell git log --format='\n%aN <%aE>' | LC_ALL=C.UTF-8 sort -uf)" >> $@
+
+.PHONY: client
+client:
```

Comment:
We already have these in `./api/makefile` - can we use those for now? I can make a PR moving those make tasks back to the root directory.